build.rs

```diff
@@ -5,29 +5,31 @@ use std::io::Write;
 use std::path::Path;
 
 fn main() {
-    let pointer_width = env::var("CARGO_CFG_TARGET_POINTER_WIDTH");
-    let u64_digit = pointer_width.as_ref().map(String::as_str) == Ok("64");
+    let ptr_width = &*env::var("CARGO_CFG_TARGET_POINTER_WIDTH").unwrap();
+
+    let u64_digit = ["64", "128"].contains(&ptr_width);
```

Comment:
FWIW, I do agree that 128-bit targets should use `u64_digit` -- that's a positive change we can make. (even though there are no such targets yet, AFAIK)

Reply:
I'll create a new PR to reflect what you mentioned.